requirements/supported_libs.txt

```diff
@@ -1,5 +1,5 @@
-opacus
+opacus==0.12.0
```

Comment:
This is needed because in version 0.13.0 ```opacus``` uses ```torchvision 0.9.0``` which needs ```torch 1.8``` - Created a subsequent [ticket](https://github.com/OpenMined/PySyft/issues/5291) for adding ```opacus``` back when we have ```torch 1.8``` support.

**IMPORTANT**: I saw that we install all the supported libs for every torch version -- I think we should have only one workflow when we test them -- if we continue to do this we would end up testing only torch1.8 (if we solve the torchvision depedency by adding torch1.8 to syft, then ```opacus``` would install each time this version of torch)

Reply:
We could move to testing the more complex set of versions less frequently like nightly or weekly.

Reply:
The problem with this I think it is that we should start pinning versions.

Reply:
Seems like this is purely to work around the MNIST dataset issue that happened the other day, I think we can try `--no-deps` instead and still use the latest opacus with older torch. It works locally in my branch, lets see what CI looks like.